src/Nimiq/BinaryCoding.h

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <stddef.h>
+#include <stdint.h>
+#include <vector>
+
+#define CAST32(x)    ((uint32_t)(x))
+#define CAST64(x)    ((uint64_t)(x))
```

Comment:
In C++ it's convention to use `static_cast<uint32_t>(x)`. Also no need for a macro.

Reply:
Good point. Maybe this should be changed in [Bitcoin/BinaryCoding.h](https://github.com/TrustWallet/wallet-core/blob/master/src/Bitcoin/BinaryCoding.h) too, then.

Reply:
Agreed.